src/hello-world/hello-world.component.spec.ts

```diff
@@ -0,0 +1,50 @@
+import {DebugElement} from '@angular/core';
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+import {By} from '@angular/platform-browser';
+import {BrowserDynamicTestingModule, platformBrowserDynamicTesting} from '@angular/platform-browser-dynamic/testing';
+
+import {HelloWorldComponent} from './hello-world.component';
+import {HelloWorldModuleNgSummary} from './hello-world.module.ngsummary';
+
+// TODO(alexeagle): this helper should be in @angular/platform-browser-dynamic/testing
+try {
```

Comment:
nit: Do you think a better pattern to establish would be to move this logic to a beforeAll block?

Reply:
Hmm, I don't have any idea what is the best practice, maybe Keen has an opinion?

Reply:
I think the example Angular app within google3 was the Google specific best practice (it had it as I described). Of course since that was just Google it could always be changed for the public specific style guide